quarto/intro.qmd

```diff
@@ -6,7 +6,24 @@
 
 ## Authoring (Estelle / Martin)
 
-### Render vs preview (Guilherme / Estelle)
+### Render vs preview
+
+The `quarto render` and `quarto preview` commands are used to generate output from a Quarto (.qmd) document. 
+
+The `quarto render` command generates output in all formats specified in your YAML header (e.g. pdf,html,word), while the `quarto preview` command only generates output in a format suitable for viewing in a web browser.
```

Comment:
The `quarto render` command generates output in all formats specified in your YAML header (e.g. pdf,html,word) or in your command line with the option --to, while the `quarto preview` command only generates output in a format suitable for viewing in a web browser.

Reply:
Thanks! Added your suggestion to the text https://github.com/NBISweden/Training-Tech-shorts/pull/41/commits/89ac5e6ddba0104a5496b68d8a052a384bfd6334